lib/proofer/v2/result.rb

```diff
@@ -10,10 +10,12 @@ def initialize(errors: {}, messages: Set.new, context: {}, exception: nil)
       @exception = exception
     end
 
+    # rubocop:disable Style/OptionalArguments
```

Comment:
We should open an issue to remember to fix this.

Reply:
To change the rule or to update the method to adhere to the rule?

Reply:
Change the method to adhere to the rule. We should only disable Rubocop offenses if there's a compelling reason. Ideally, we would use keyword arguments to avoid confusion. https://robots.thoughtbot.com/ruby-2-keyword-arguments